### PR TITLE
107 feat add john to the team section

### PR DESCRIPTION
**Pull request by hollandjg**
Adding me to team site, first attempt at generating pull request to see preview.

### CODE REVIEW

File: about/people.yml [hunk 1]
```diff
@@ -314,3 +314,15 @@ data:
...
+    team: Advanced Research Computing
+    subteam: Data & Visualization
+    title: Senior Data Scientist
+    github_username: hollandjg
+    brown_directory_uuid: c58cefc3-dc27-408f-ae62-fa098803341d
+    bio: John is a data scientist at the CCV, who is passionate about climate, 
+      Earth observation and sustainability. Prior to starting at the CCV, John studied physics,  
+      received his doctorate for research in astronomy, 
+      and worked for a boutique management and IT consultancy in Germany.
+    image: john_main.jpg
```
Comment (mirestrepo) [inline about/people.yml:328]: The images are not showing up in the preview link. I think you may just need to make the files `jpg` instead of `jpeg`

**Review by mirestrepo — COMMENTED** [review]
Why did you close it! This is great. I think just the extension of the images needs to change. Seems like we may assume `jpg`

**Comment by hollandjg** [comment]
I think that should do it. I'll check out the result on Monday!

**Review by paulstey — APPROVED** [review]
This looks good to me; thank you, @hollandjg!!

**Pull request merged**